extract-metadata.py

```diff
@@ -20,18 +20,44 @@ def extract_md(filename):
 
 def extract_xml(filename):
     parser = xml.sax.make_parser()
-    handler = PIHandler()
+    handler = MdHandler()
     parser.setContentHandler(handler)
     parser.parse(filename)
-    return handler.pis
+    return handler.md
```

Comment:
```suggestion
    return handler.metadata
```

Similar naming throughout.